src/data/fintype/fin.lean

```diff
@@ -45,4 +47,24 @@ begin
   { rintro ⟨i, hi, rfl⟩, simpa },
 end
 
+lemma card_filter_univ_succ' (p : (fin (n + 1)) → Prop) [decidable_pred p] :
```

Comment:
These parens are redundant, I already suggested removing them in a previous review.
```suggestion
lemma card_filter_univ_succ' (p : fin (n + 1) → Prop) [decidable_pred p] :
```